README.dev.md

```diff
@@ -0,0 +1,95 @@
+## How to run tests
```

Comment:
FYI: I usually call it HACKING.md, however I don't whether it is widely spread.

Feel free to accept it if you like it or ignore.

Reply:
I've seen several README.dev.md examples, so I think it's legal.

Reply:
Okay.